Sources/PayPalNativePayments/PayPalNativeCheckoutClient.swift

```diff
@@ -43,6 +43,7 @@ public class PayPalNativeCheckoutClient {
         request: PayPalNativeCheckoutRequest,
         presentingViewController: UIViewController? = nil
     ) async {
+        // NEXT_MAJOR_VERSION: - Change to non-async
```

Comment:
I think we want to move this above `/// Present PayPal Paysheet and start a PayPal transaction`

Reply:
Addressed here: dd91cffd9ccb5cd849acaf586d5379a2c82d3dff